spec/emerald/compiler_spec.rb

```diff
@@ -17,4 +17,24 @@
       expect(compiled_lisp).to eq("")
     end
   end
+
+  context "when the contents of the file define a variable" do
+    it "parses the lisp and returns an assigned global variable" do
+      lisp_file_contents = "(def y (- 1 6))"
+      compiled_lisp = Emerald::Compiler.new(lisp_file_contents).compile
+      expect(compiled_lisp).to eq("y = 1.0 - 6.0")
```

Comment:
Hmm I'm not sure I understand you here. Do you mean that they're not global variables by Ruby's standard because global variables in Ruby are like `$variable`?